nadlogar/problems/models/kvadratna.py

```diff
@@ -0,0 +1,484 @@
+#from generiranje import Naloga, preveri
+import random
+import sympy
+from .linearna import skozi_tocki, seznam_polovick, seznam_tretinj
+from django.db import models
+
+from .meta import GeneratedDataIncorrect, Problem
+
+
+
+def nicelna_oblika(od=-5, do=5, risanje=False):
+    """
+    Vrne naključno kvadratno funkcijo v ničelni obliki.
+    :param od: najmanjša možna vrednost za ničlo funkcije
+    :param do: največja možna vrednost za ničlo funkcije
+    :param risanje: če fukcijo potrebujejmo za risanje, izbere lepši vodilni koeficient
```

Comment:
Z Gabrijelom sva se odločila, da bova komentarje pod definicijami funkcij skrajšala. Smiselno je pustiti kak primer delovanja funkcije, izbriši pa vrtice s `:param ... , :return ....`

Reply:
Zdi se mi, da je tudi razlago parametrov in tega, kar funkcija vrača, smiselno pustiti za boljšo preglednost in razumevanje pomožnih funkcij.